SystemInformer/modprv.c

```diff
@@ -515,13 +515,18 @@ VOID PhModuleProviderUpdate(
     PPH_MODULE_PROVIDER moduleProvider = (PPH_MODULE_PROVIDER)Object;
     PPH_LIST modules;
     ULONG i;
+    PH_FORMAT format[2];
+    SIZE_T returnLength;
 
     // If we didn't get a handle when we created the provider,
     // abort (unless this is the System process - in that case
     // we don't need a handle).
     if (!moduleProvider->ProcessHandle && moduleProvider->ProcessId != SYSTEM_PROCESS_ID)
         goto UpdateExit;
 
+    PhInitFormatS(&format[0], L"0x");
+    PhInitFormatIXPadZeros(&format[1], 0);
```

Comment:
Move this down to where it is used, please - then you can remove the assignment to `UIntPtr`.

Reply:
Are you saying to move only the `PhInitFormatIXPadZeros` or both format functions (& the variables `format`/`returnLength`)?

Reply:
Move both down before `PhFormatToBuffer`. This way you don't have to look up to hunt for the formatting that is going to be applied to the `PhFormatToBuffer(.., 2..`.